docs/Plugins/gitlab.md

```diff
@@ -13,6 +13,9 @@ This plugin collects GitLab data through [API](https://docs.gitlab.com/ee/api/).
 - How long does it take for your codes to get merged?
 - How much time is spent on code review?
 
+## Supported Versions
+Available for GitLab Cloud, Community Edition 11+
```

Comment:
Available for GitLab Cloud, Community Edition 11+. Check [this doc](relative url of the supported data sources) for more details.